scripts/ci/upload-release

```diff
@@ -28,14 +28,12 @@ SCRIPT_PATH="$(cd $(dirname "$0")/$(dirname "$(readlink "$0")") && pwd)"
 # project root for this file
 PROJECT_ROOT="$( cd "$( echo "${SCRIPT_PATH}" | sed s+/scripts/ci++)" && pwd )"
 
-# get correct DC/OS target (=base branch!) from .dcosrc file
-PKG_TARGET=$(python -c "import sys, json; \
-  dcosrc = json.load(open('$PROJECT_ROOT/.dcosrc')); \
-  sys.stdout.write(str(dcosrc['base']));")
-
 # next release version (passed in by semantic-release)
 PKG_VERSION="${1}"
 
+# get correct DC/OS target (=base branch!) (passed in by semantic-release)
```

Comment:
Why can't we use base branch? How would it look like on master?

Reply:
our branch is named `release/1.12` while dcos branch is named just `1.12` - I had a fance regex before, but that was too complicated in the end…

Reply:
but we could read this data from `.dcosrc` can't we?

Reply:
My idea was to get rid of this file and have all the information in `.releaserc` 🤔

Reply:
@nLight anything else preventing you from ✅?

Reply:
Did we capture all the improvements in a JIRA?

Reply:
nope, shall I create one?

Reply:
would appreciate it

Reply:
DCOS-43338

Reply:
I'm sorry I mean a JIRA to further improve what we've discussed here. Like consolidating variables for a release number etc.